game-core/src/main/java/games/strategy/net/IMessenger.java

```diff
@@ -16,11 +16,6 @@
    */
   void send(Serializable msg, INode to);
 
-  /**
-   * Send a message to all nodes.
-   */
-  void broadcast(Serializable msg);
-
```

Comment:
I'm surprised this method isn't used. How does chat broadcast the messages if not via this method?

Reply:
Nullable state is used:
```
@Getter
@AllArgsConstructor
public class MessageHeader {
  // to can be null if we are a broadcast
  @Nullable
  private final INode to;
  // from can be null if the sending node doesnt know its own address
  @Nullable
  private final INode from;
  private final Serializable message;
}
```

Reply:
Ok wow.
Legacy code strikes again. 😥 
I'm not even surprised anymore.